src/System.IO.FileSystem/src/Resources/Strings.resx

```diff
@@ -174,6 +174,9 @@
   <data name="Argument_EmptyPath" xml:space="preserve">
     <value>Empty path name is not legal.</value>
   </data>
+  <data name="Argument_FileNotResized" xml:space="preserve">
+    <value>The file system was not able to change the size of the file.</value>
```

Comment:
Is the condition it checks only an indication about not being able to, or could it just indicate a "temporary" failure?  IOW, would past-tense be more appropriate here:
`"The stream's length could not be changed."`?